scripts/scons_plugin/config.py

```diff
@@ -457,7 +457,10 @@ def AddPkgConfigDependency(context, package, flags):
         context.Result('pkg-config not available')
         return False
 
-    cmd = '%s %s --silence-errors %s' % (quote(pkg_config), package, flags)
+    cmd = ''
+    if config_path and os.path.isdir(config_path):
+        cmd = 'env PKG_CONFIG_PATH={} '.format(quote(config_path))
+    cmd += '{} {} --silence-errors {}'.format(quote(pkg_config), package, flags)
```

Comment:
nit: we mostly use %s formatting in scons, could you use it here too for consistency?

Reply:
`format()` is always better and safer; see
- https://docs.python.org/2/library/string.html#format-examples
- https://stackoverflow.com/a/5082482